src/comments/comment.service.ts

```diff
@@ -0,0 +1,360 @@
+    }
+
+    await this.commentsRepository.remove(comment);
+  }
+
+  async commentAnswer(answerId: number, content: string): Promise<CommentDto> {
+    const answer = await this.answersRepository.findOneBy({ id: answerId });
+
+    if (!answer) {
+      throw new Error('Answer not found');
+    }
+
+    const comment = new Comment();
+    comment.content = content;
+    comment.commentableType = 'answer';
+    comment.commentableId = answer.id;
+
+    const createdComment = await this.commentsRepository.save(comment);
+
+    const commentDto: CommentDto = {
+      id: createdComment.id,
+      content: createdComment.content,
+      commentableId: comment.commentableId,
+      commentableType: comment.commentableType,
+      user: comment.user,
+      quote: {
+        quote_id: null,
+        quote_user: null,
+      },
+      agree_count: 0,
+      disagree_count: 0,
+      agree_type: 0,
+      created_at: comment.createdAt.getTime(),
+    };
+
+    return commentDto;
+  }
+
+  async commentQuestion(
+    questionId: number,
+    content: string,
+  ): Promise<CommentDto> {
+    const question = await this.questionsRepository.findOneBy({
+      id: questionId,
+    });
+
+    if (!question) {
+      throw new Error('Answer not found');
+    }
+
+    // 创建评论
+    const comment = new Comment();
+    comment.content = content;
+    comment.commentableType = 'question';
+    comment.commentableId = question.id;
+    const createdComment = await this.commentsRepository.save(comment);
+    const commentDto: CommentDto = {
+      id: createdComment.id,
+      content: createdComment.content,
+      commentableId: comment.commentableId,
+      commentableType: comment.commentableType,
+      user: comment.user,
+      quote: {
+        quote_id: null,
+        quote_user: null,
+      },
+      agree_count: 0,
+      disagree_count: 0,
+      agree_type: 0,
+      created_at: comment.createdAt.getDate(),
+    };
+    return commentDto;
+  }
+
+  async commentComment(
+    commentId: number,
+    content: string,
+  ): Promise<CommentDto> {
+    const Comments = await this.questionsRepository.findOneBy({
+      id: commentId,
+    });
+
+    if (!Comments) {
+      throw new Error('Answer not found');
+    }
+
+    const comment = new Comment();
+    comment.content = content;
+    comment.commentableType = 'question';
+    comment.commentableId = Comments.id;
+
+    const createdComment = await this.commentsRepository.save(comment);
+
+    const commentDto: CommentDto = {
+      id: createdComment.id,
+      content: createdComment.content,
+      commentableId: comment.commentableId,
+      commentableType: comment.commentableType,
+      user: comment.user,
+      quote: {
+        quote_id: null,
+        quote_user: null,
+      },
+      agree_count: 0,
+      disagree_count: 0,
+      agree_type: 0,
+      created_at: comment.createdAt.getDate(),
+    };
+
+    return commentDto;
+  }
+
+  async getCommentDetail(
```

Comment:
There is a mistake in `getCommentDetail` API definition. Please mention that `getCommentDetail` has nothing to do with page logic. It just takes `comment_id` and give the commentDto for detail. Please revise cotroller, service and API.yml.